insonmnia/locator/app.go

```diff
@@ -0,0 +1,102 @@
+package locator
+
+import (
+	"crypto/ecdsa"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/noxiouz/zapctx/ctxlog"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	"github.com/docker/libkv"
+	"github.com/docker/libkv/store"
+	"github.com/docker/libkv/store/boltdb"
+
+	pb "github.com/sonm-io/core/proto"
+	"github.com/sonm-io/core/util"
+
+	engine "github.com/sonm-io/core/insonmnia/locator/storage/libkv"
```

Comment:
How does "engine" alias make it more clear?

Reply:
It's here not to make it clear, but to make it work, because there are two different libkv packages:
	* "github.com/docker/libkv"
	* engine "github.com/sonm-io/core/insonmnia/locator/storage/libkv"